app/src/main/java/org/gdg/frisbee/android/fragment/SettingsFragment.java

```diff
@@ -147,28 +150,34 @@ public void onNotFound(String key) {
 
         CheckBoxPreference prefGoogleSignIn = (CheckBoxPreference) findPreference(PrefUtils.SETTINGS_SIGNED_IN);
         if (prefGoogleSignIn != null) {
-            prefGoogleSignIn.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
-                @Override
-                public boolean onPreferenceChange(Preference preference, Object newValue) {
-                    boolean signedIn = (Boolean) newValue;
-                    if (signedIn) {
-                        if (mGoogleApiClient.isConnected()) {
-                            disconnectGoogleApiClient();
-                        }
-                        PrefUtils.setSignedIn(getActivity());
-                        createConnectedGoogleApiClient();
-                    } else {
-                        if (mGoogleApiClient.isConnected()) {
-                            Games.signOut(mGoogleApiClient);
-                            Plus.AccountApi.clearDefaultAccount(mGoogleApiClient);
-                            disconnectGoogleApiClient();
+            if (GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(getActivity())
+                == ConnectionResult.SUCCESS) {
+                prefGoogleSignIn.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
+                    @Override
+                    public boolean onPreferenceChange(Preference preference, Object newValue) {
+                        boolean signedIn = (Boolean) newValue;
+                        if (signedIn) {
+                            if (mGoogleApiClient.isConnected()) {
+                                disconnectGoogleApiClient();
+                            }
+                            PrefUtils.setSignedIn(getActivity());
+                            createConnectedGoogleApiClient();
+                        } else {
+                            if (mGoogleApiClient.isConnected()) {
+                                Games.signOut(mGoogleApiClient);
+                                Plus.AccountApi.clearDefaultAccount(mGoogleApiClient);
+                                disconnectGoogleApiClient();
+                            }
+                            PrefUtils.setLoggedOut(getActivity());
+                            createConnectedGoogleApiClient();
                         }
-                        PrefUtils.setLoggedOut(getActivity());
-                        createConnectedGoogleApiClient();
+                        return true;
                     }
-                    return true;
-                }
-            });
+                });
+            } else {
+                PreferenceScreen root = (PreferenceScreen) findPreference(PrefUtils.SETTINGS_ROOT);
```

Comment:
:+1: I was thinking to disable it and set the description to something like this: "Play Services not available"